react/react1/week3/todo-app-week3/src/components/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import TodoRow from './TodoRow';
+
+const TodoList = ({ todos, setTodos }) => {
+  return (
+    <div className='todo-container'>
+      <p className='todo-list'>
+        {todos.map((item) => (
+          <TodoRow
+            key={item.id}
+            item={item}
```

Comment:
i would actually send the whole item object here instead of doing a prop for item.description etc. This way you will avoid sending as many props which makes your code more maintainable. When we have components with a lot of props it gets hard to maintain and understand what every single prop is doing.